src/main/java/com/gmail/nossr50/listeners/EntityListener.java

```diff
@@ -313,29 +348,28 @@ public void onEntityDamageByEntity(EntityDamageByEntityEvent event) {
         Entity defender = event.getEntity();
         Entity attacker = event.getDamager();
 
-        if(WorldGuardUtils.isWorldGuardLoaded())
-        {
-            if(attacker instanceof Player) {
+        if (WorldGuardUtils.isWorldGuardLoaded()) {
+            if (attacker instanceof Player) {
 
-                if(!WorldGuardManager.getInstance().hasMainFlag((Player) attacker)) {
+                if (!WorldGuardManager.getInstance().hasMainFlag((Player) attacker)) {
                     return;
                 }
 
-            } else if(attacker instanceof Projectile) {
+            } 
+            else if (attacker instanceof Projectile) {
 
                 Projectile projectile = (Projectile) attacker;
 
-                if(projectile.getShooter() instanceof Player) {
-                    if(!WorldGuardManager.getInstance().hasMainFlag((Player) projectile.getShooter())) {
-                        return;
-                    }
+                if (projectile.getShooter() instanceof Player 
+                        && !WorldGuardManager.getInstance().hasMainFlag((Player) projectile.getShooter())) {
```

Comment:
Nested if conditions are easier to read, if you want to join multiple if statements put them in a method to increase readability